pageserver/src/tenant/timeline.rs

```diff
@@ -2089,14 +2089,12 @@ impl Timeline {
         }
 
         info!(
-            "GC completed removing {} layers, cuttof {}",
+            "GC completed removing {} layers, cutoff {}",
             result.layers_removed, new_gc_cutoff
         );
+
         if result.layers_removed != 0 {
-            fail_point!("gc-before-save-metadata", |_| {
-                info!("Abnormaly terinate pageserver at gc-before-save-metadata fail point");
```

Comment:
Sorry, the idea of this fail point was to terminate pageserver.
As far as I noticed you configure this failpoit in test_gc_cutoff with "exit" task.
There is no such task according to the documentation: https://docs.rs/failpoints/0.2.0/failpoints/fn.cfg.html
But even if this documentation is not up-to-date and this task is really supported, I prefer to kill pageserver  and do not let him perform normal on-exit cleanup.

Reply:
> As far as I noticed you configure this failpoit in test_gc_cutoff with "exit" task.
> There is no such task according to the documentation: https://docs.rs/failpoints/0.2.0/failpoints/fn.cfg.html

It's a Neon addition,see https://github.com/neondatabase/neon/blob/4d1e48f3b9a4b7064787513fd2c455f0001f6e18/pageserver/src/http/routes.rs#L762

Reply:
I moved this change to https://github.com/neondatabase/neon/pull/2655